packages/tasks/src/local-apps.ts

```diff
@@ -34,6 +34,12 @@ export type LocalApp = {
 			 */
 			deeplink: (model: ModelData) => URL;
 	  }
+	| {
+			/**
+			 * If the app supports opening through their website
+			 */
+			weblink?: (mode: ModelData) => URL;
+	  }
```

Comment:
it could be the same no? ie. i was thinking of using `deeplink()` for it (conceptually speaking it's the same)

Reply:
yes I can be the same